cloudinary_cli/modules/sync.py

```diff
@@ -348,16 +364,19 @@ def _handle_unique_remote_files(self):
 
             # Each batch is further chunked by a deletion batch size that can be specified by the user.
             for deletion_batch in chunker(batch, self.deletion_batch_size):
-                res = api.delete_resources(deletion_batch, invalidate=True, resource_type=attrs[0], type=attrs[1])
-                num_deleted = Counter(res['deleted'].values())["deleted"]
-                if self.verbose:
-                    print_json(res)
-                if num_deleted != len(deletion_batch):
-                    # This should not happen in reality, unless some terrible race condition happens with the folder.
-                    failed = [f"{file}: {reason}" for file, reason in res['deleted'].items() if reason != "deleted"]
-                    logger.error("Failed deletes:\n{}".format("\n".join(failed)))
+                if self.dry_run:
+                    logger.info(f"Dry run mode enabled. Would delete {len(deletion_batch)} resources: {deletion_batch}")
```

Comment:
I would prefer not to put the main logic of the function under `else` statement, to improve readability.

It would be better to add `continue` here and keep the rest as it was before.